provider/oci/provider.go

```diff
@@ -261,6 +275,16 @@ func (e *EnvironProvider) Open(params environs.OpenParams) (environs.Environ, er
 		return nil, errors.New("compartment-id may not be empty")
 	}
 
+	addressSpace := cfg.addressSpace()
+	if _, ipNET, err := net.ParseCIDR(*addressSpace); err == nil {
+		size, _ := ipNET.Mask.Size()
```

Comment:
I think the error is probably confusing for users.
A subnet of size "/8" is *larger* than a subnet of size "/16".
If I read it correctly, ipnet.Size() returns the "number of leading ones", so it is the /X in the CIDR.
Are you wanting to say "I must have at least a /16, which means a /8 is ok", or are you saying "you can't include a larger subnet than /16" ?
Maybe if we rephrase it to:
"configured subnet %s/%d is not large enough. please configure a subnet in the range of /8 to /16"
?

Suggestions welcome, I just know that the first time I read over this I was certainly confused.

Reply:
Changed. I have no strong opinions on the phrasing of the error. Whatever you think is less confusing, is fine with me.